starter-code/index.html

```diff
@@ -10,5 +10,50 @@
   <title>Ironhack cart</title>
 </head>
 <body>
+
+  <div class="header">
+    <h1>Ironhack Merchandising Shop</h1>
+  </div>
+
+
+  <div class="item">
+    <div id="itemName"><span>Iron Bubblehead</span></div>
+    <div id="bubblePrice" class="price"><span>$25</span></div>
+
+    <div class="quantity">
+      <label for="quantity">QTY: </label>
+      <input name="quantity" type="number" class="quantity-input">
```

Comment:
You can make the default value equal to zero so if a user doesn't add a quantity then you won't get NaN
```
<input value="0" name="quantity" type="number" class="quantity-input">
```